src/faq/index.md

```diff
@@ -49,6 +49,16 @@ There are several community channels available for sharing experiences built A-F
 
 We're easiest to reach on the [A-Frame Slack channel](https://aframevr.slack.com/), via Twitter at [@aframevr](https://twitter.com/aframevr), or by filing issues against the A-Frame [GitHub repo](http://github.com/aframevr/aframe). We love questions, feedback, bug reports and pull requests!
 
+## How is A-Frame different from VRML?
+
+A-Frame is a JavaScript framework—not a web standard. It is built on top of the WebGL API, which is widely supported on modern browsers for desktop and mobile devices. That means it works for developers and users _today_. Browser vendors do not need to know A-Frame exists, or implement support for it. 
+
+This is in contrast to the old approach to implementing new functionality in the web platform, wherein proposal, ratification and implementation of new web standards was a multi-year process with no gaurantee of successful outcome.
```

Comment:
to be consistent with usage elsewhere, add a serial comma: "ratification, and implementation"